core/src/wallet/tezos/explorers/ExternalTezosLikeBlockchainExplorer.cpp

```diff
@@ -137,12 +137,17 @@ namespace ledger {
 
         FuturePtr<TezosLikeBlockchainExplorer::TransactionsBulk>
         ExternalTezosLikeBlockchainExplorer::getTransactions(const std::vector<std::string> &addresses,
-                                                             Option<std::string> fromBlockHash,
+                                                             Option<std::string> offset,
```

Comment:
`offset` is not really a good name here. What offset?

Reply:
It's the `offset` related to pagination, our explorers paginate according to block hashes and tzstats do it according to an `limit` and `offset` that you append as params to your query.
And BTW this is pretty straightforward when you look at the code of `getTransactions`.